Makefile

```diff
@@ -63,20 +74,79 @@ check_headers:
 	@echo "Checking file headers"
 	@python test/verify_boilerplate.py
 
+# Integration tests
+.PHONY: test_integration
+test_integration:
+	bundle install
+	bundle exec kitchen create
+	bundle exec kitchen converge
+	bundle exec kitchen converge
+	bundle exec kitchen verify
+	bundle exec kitchen destroy
+	cd test/fixtures/networks/ && terraform destroy -auto-approve && cd -
```

Comment:
Unfortunately not. The lifecycle hook actually runs at the end of every `test-kitchen` suite, not at the end of all of the suites. None of the lifecycles documented (https://github.com/test-kitchen/test-kitchen/blob/master/RELEASE_NOTES.md) will run solely before or after the entire set of suites.

We're able to get away with this in `pre_converge` because applying the fixtures when they already exist is an idempotent (and relatively quick) operation -- and one could argue that it makes sense that if a test case, somehow, messes with the fixture networks, it is sensible to "repair" them.

Reply:
Ah, okay. Makes sense!